server/src/Model/Message/MessageSchema.js

```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+import { MESSAGE_SCHEMA } from '../../Constants/constants';
+
+const messageSchema = new Schema({
+  from: String,
+  to: String,
+  message: String,
+  workspace: String,
+  channel: String,
+  createdAt: Date,
```

Comment:
you can use mongoose timestamps for this https://mongoosejs.com/docs/guide.html#timestamps